website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -1076,6 +1076,32 @@ You can specify the following parameters to configure ingress gateway configurat
               respected, a L7 protocol must be defined in the \`protocol\` field.
               If not specified, it uses the default value. For example, 1024 for Envoy proxy.`,
         },
+        {
+          name: 'PassiveHealthCheck',
+          type: 'PassiveHealthCheck: <optional>',
+          description:
+            'Passive health checks are used to remove hosts from the upstream cluster which are unreachable or are returning errors.',
+          children: [
+            {
+              name: 'interval',
+              type: 'int: <optional>',
+              description:
+                "The time in nanosecond between checks. Each check will cause hosts which have exceeded `max_failures` to be removed from the load balancer, and any hosts which have passed their ejection time to be returned to the load balancer. If not specified, it uses the default value. For example, 10s for Envoy proxy.",
+            },
+            {
+              name: 'max_failures',
+              type: 'int: <optional>',
+              description:
+                'The number of consecutive failures which cause a host to be removed from the load balancer. If not specified, it uses the default value. For example, 5 for Envoy proxy.',
```

Comment:
```suggestion
                'The number of consecutive failures that cause a host to be removed from the load balancer. If not specified, Consul uses the proxy's default value. For example, `5` for Envoy proxy.',
```
Just FYI, use "that" to add essential information and "which" preceded by a comma to add a non-essential phrase. Also, I replaced "it" to make it clearer that Consul is the thing doing the action. If that is incorrect, we should be clear about _what_ "uses the proxy's default value".